### DISCUSSION TITLE
how to recover data rejected by Elasticsearch

### DISCUSSION THREAD

**Discussion by tshepang** [vectordotdev/vector#15190] [Q&A] [answered]
An example error would be...
```json
{"type":"illegal_argument_exception","reason":"Limit of total fields [1000]...", ...
```
Is there an easy way to view such rejected data, similar to how one can do a `reroute_dropped` of remap transform?

**Comment by jszwedko** [comment, marked as answer]
Unfortunately not yet, but we are working an adding the ability to route failed events from sinks. See https://github.com/vectordotdev/vector/pull/14708 for the proposed UX.
